test/mongoose_push_web/test_helper.exs

```diff
@@ -1,2 +1,6 @@
 {:ok, _} = Application.ensure_all_started(:mongoose_push)
 ExUnit.start(capture: true)
+
+defmodule Mocks do
```

Comment:
Module here is redundant and actually may even cause issues. Let's never put modules in scripts (`.exs`).

Reply:
corrected, but the same was already done in `test/unit/test_helper.exs`